addons/website/static/src/snippets/s_chart/000.js

```diff
@@ -116,41 +111,23 @@ const ChartWidget = publicWidget.Widget.extend({
         const canvas = this.el.querySelector('canvas');
         window.Chart.Tooltip.positioners.custom = (elements, eventPosition) => eventPosition;
         this.chart = new window.Chart(canvas, chartData);
-        return this._super.apply(this, arguments);
-    },
+    }
 
-    willStart: async function () {
-        await loadBundle("web.chartjs_lib");
-    },
-    /**
-     * @override
-     * Discard all library changes to reset the state of the Html.
-     */
-    destroy: function () {
-        if (this.chart) { // The widget can be destroyed before start has completed
-            this.chart.destroy();
-            this.el.querySelectorAll('.chartjs-size-monitor').forEach(el => el.remove());
-        }
-        this._super.apply(this, arguments);
-    },
-
-    //--------------------------------------------------------------------------
-    // Private
-    //--------------------------------------------------------------------------
+    destroy() {
+        this.chart.destroy();
```

Comment:
what if the interaction is destroyed before willStart is complete? => boom, that's what happens!